doc/manual/meson.build

```
@@ -0,0 +1,75 @@
+if get_option('doc')
+  ignore_headers = [
+    'private.h',
+    'pkcs11.h',
+    'conf.h',
+    'debug.h',
+    'dict.h',
+    'filter.h',
+    'log.h',
+    'mock.h',
+    'modules.h',
+    'pkcs11.h',
+    'pkcs11i.h',
+    'pkcs11x.h',
+    'private.h',
+    'client.h',
+    'proxy.h',
+    'rpc.h',
+    'rpc-message.h',
+    'util.h',
+    'virtual.h',
+    'virtual-fixed.h',
+    'array.h',
+    'compat.h'
+  ]
+
+  content_files = [
+    'p11-kit-config.xml', 'p11-kit-sharing.xml',
+    'p11-kit-devel.xml',
+    'p11-kit-proxy.xml',
+    'p11-kit-trust.xml',
+    'p11-kit-remoting.xml',
+    'p11-kit.xml',
+    'pkcs11.conf.xml',
+    'trust.xml',
+    'annotation-glossary.xml'
+  ]
+
+  version_conf = configuration_data()
+  version_conf.set('VERSION', meson.project_version())
```

Since meson 0.49 you can use dicts for `configuration_data` objects only using `set`. So this would just be:

```suggestion
  version_conf = {'VERSION': meson.project_version()}
```

And being only one line, why don't just write it in the `configure_file` call?

```suggestion
  configure_file(
    input : 'version.xml.in',
    output : 'version.xml',
    configuration : {'VERSION': meson.project_version()})
```

Finally, you can also use `@BASENAME@` as output value, that removes the last extension:


```suggestion
  configure_file(
    input : 'version.xml.in',
    output : '@BASENAME@',
    configuration : {'VERSION': meson.project_version()})
```

---

Nice tip, rewritten with it.